### DISCUSSION TITLE
Any way to "unpublish / publish" in the admin portal?

### DISCUSSION THREAD

**Discussion by oscadev** [pocketbase/pocketbase#761] [Q&A] [answered]
Is there a way to denote what entries not to return in an API call, in the admin portal other than by deleting the entries?

**Comment by ganigeorgiev** [comment, marked as answer]
You can do that on your own by adding a "published" `bool` field to your collection and for the List and View API rules use something like:

```
published = true
```
to return only the published records.

**Reply by oscadev** [reply]
Nice. Thanks!

**Reply by ganigeorgiev** [reply]
@oscadev Please note that the above will allow anyone, including unregistered users, to list and view the "published" records of your collection. If that's not what you want, make sure to append the correct rule to further restrict the access, eg:

```
published = true && @request.user.id != ""
```
(this will return only the records when they are marked as "published" and the user is authenticated).

The documentation is currently a bit lacking on this, but if you haven't already, you could check https://pocketbase.io/docs/manage-collections/#collection-api-rules
